### DISCUSSION TITLE
suggest float window drawn incorrectly if border is set to true(v0.0.82)

### DISCUSSION THREAD

**Discussion by benzwt** [neoclide/coc.nvim#3993] [Q&A] [answered]
For v0.0.82, the right border of the suggest float window is thicker than the left border, if the suggest.floatConfig.border is set to true.
The side effect is, part of the preview window is shadowed by the suggest float window.

### CocInfo
```
## versions

vim version: VIM - Vi IMproved 9.0 9000124
node version: v12.22.9
coc.nvim version: 0.0.82-09b93169 2022-08-02 21:04:13 +0800
coc.nvim directory: /home/anoymous/.vim/plugged/coc.nvim
term: tmux
platform: linux

## Log of coc.nvim

2022-08-04T19:47:52.175 INFO (pid:136859) [plugin] - coc.nvim initialized with node: v12.22.9 after 157ms
2022-08-04T19:47:55.342 INFO (pid:136859) [attach] - receive notification: showInfo []


```

### suggest float window, border=true, preview window is shadowed 
![suggest_float_border_true.png](https://raw.githubusercontent.com/benzwt/sharing/main/.github/images/suggest_float_border_true.png)
### suggest float window, border=false
![suggest_float_border_false.png](https://raw.githubusercontent.com/benzwt/sharing/main/.github/images/suggest_float_border_false.png)

**Comment by chemzqm** [comment]
Need the result of `:CocInfo`

**Reply by benzwt** [reply]
here is the CocInfo
```
## versions

vim version: VIM - Vi IMproved 9.0 9000124
node version: v12.22.9
coc.nvim version: 0.0.82-09b93169 2022-08-02 21:04:13 +0800
coc.nvim directory: /home/anoymous/.vim/plugged/coc.nvim
term: tmux
platform: linux

## Log of coc.nvim

2022-08-04T19:47:52.175 INFO (pid:136859) [plugin] - coc.nvim initialized with node: v12.22.9 after 157ms
2022-08-04T19:47:55.342 INFO (pid:136859) [attach] - receive notification: showInfo []
```

**Reply by chemzqm** [reply]
Try upgrade your coc.nvim

**Comment by chemzqm** [comment, marked as answer]
It seems you have `'ambiwidth'` set to `'double'`, try minimal vimrc.
